pkgs/by-name/th/thc-secure-delete/package.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, stdenvNoCC # x86_64-darwin and aarch64-darwin cant find C compiler
+, fetchFromGitHub
+, pkg-config
+, gcc
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "thc-secure-delete";
+  version = "3.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gordrs";
+    repo = "thc-secure-delete";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-hvWuxFkzhOSCplPtyjRtn36bIk6KdPBcpr3lAmiAyfE=";
+  };
+
+  build-systems = [ pkg-config gcc ];
```

What is this supposed to do? This isn't used anywhere by nixpkgs or (as far as I can see unless github search just didn't show me anything) the upstream project. If you need pkg-config, use this:

```suggestion
  nativeBuildInputs = [ pkg-config ];
```